kinto/core/testing.py

```diff
@@ -70,7 +70,7 @@ class FormattedErrorMixin(object):
     """
 
     def assertFormattedError(self, response, code, errno, error,
-                             message=None, info=None):
+                             message=None, info=None, details=None):
```

Comment:
It seems like we don't use this `details` variable anywhere here. Am I missing something?